odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -127,6 +127,7 @@ def test_properties_field(self):
         self.assertEqual(self.message_3.read(['attributes'])[0]['attributes'], expected)
         self.assertEqual(self.message_3.attributes, expected)
 
+    @mute_logger('odoo.fields')
```

Comment:
Commit typo: "log a message each time a user change a definition" -> changes a definition (or "each time a definition is modified", probably better)